app/scripts/modules/kubernetes/serverGroup/configure/CommandBuilder.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.kubernetes.serverGroupCommandBuilder.service', [
+  require('../../../core/cache/deckCacheFactory.js'),
+  require('../../../core/account/account.service.js'),
+  require('../../../core/naming/naming.service.js'),
+  require('../../../core/utils/lodash.js'),
+])
+  .factory('kubernetesServerGroupCommandBuilder', function (settings, $q, accountService, namingService, _) {
+    function attemptToSetValidCredentials(application, defaultCredentials, command) {
+      return accountService.listAccounts('kubernetes').then(function(kubernetesAccounts) {
+        var kubernetesAccountNames = _.pluck(kubernetesAccounts, 'name');
+        var firstKubernetesAccount = null;
+
+        if (application.accounts.length) {
+          firstKubernetesAccount = _.find(application.accounts, function (applicationAccount) {
+            return kubernetesAccountNames.indexOf(applicationAccount) !== -1;
+          });
+        }
+
+        var defaultCredentialsAreValid = defaultCredentials && kubernetesAccountNames.indexOf(defaultCredentials) !== -1;
+
+        command.credentials =
+          defaultCredentialsAreValid ? defaultCredentials : (firstKubernetesAccount ? firstKubernetesAccount : 'my-account-name');
+      });
+    }
+
+    function buildNewServerGroupCommand(application, defaults) {
+      defaults = defaults || {};
```

Comment:
You can omit this line by changing the method signature to:
```javascript
function buildNewServerGroupCommand(application, defaults = {})
```
Babel will end up converting it to exactly what's here.